modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -2943,24 +2959,26 @@ public void onExchangeDone(
     /**
      * @param grpId Group ID.
      * @param destroy Group destroy flag.
+     * @param updateMetrics Remove metrics flag.
```

Comment:
Let's name this flag `metricsEnabled` or `removeMetrics` here and all over the patch.